scripts/serial-test.sh

```diff
@@ -1,36 +1,25 @@
 #!/bin/bash
-set -e # exit on failure of any "simple" command (excludes &&, ||, or | chains)
-# prepare environment
+set -eo pipefail
+[[ -f build.tar.gz ]] && tar -xzf build.tar.gz
+cd ./build
+echo "[Killing old MongoDB]"
+$(pgrep mongod | xargs kill -9) || true
```

Comment:
`$(pgrep mongod | xargs kill -9) || true`    
is equivalent to  
`$(pgrep mongod | xargs kill -9) || :`    
The `:` character is no-op in BASH.